Src/StartMenu/StartMenuDLL/StartMenuDLL.rc

```diff
@@ -637,6 +637,10 @@ BEGIN
     IDS_MENU_USERNAME       "User name text"
     IDS_MENU_USERNAME_TIP   "Enter the text you want to see in the user-name portion of the menu (for skins that show the user name)"
     IDS_PIC_COMMAND         "User picture command"
+    IDS_ENABLE_ACCELERATORS "Enable accelerators"
```

Comment:
@bonzibudd 
Do you think these texts are understandable?
Feel free to propose any changes that will make them better.

Reply:
Yes, I think that works. That way it's clear that the alt key option is dependent on the prior.